WordPress/src/main/java/org/wordpress/android/localcontentmigration/LocalContentEntity.kt

```diff
@@ -0,0 +1,35 @@
+package org.wordpress.android.localcontentmigration
+
+import org.wordpress.android.fluxc.model.PostModel
+
+enum class LocalContentEntity(private val isSiteContent: Boolean = false) {
```

Comment:
After our more recent discussions, with the shift in approach to preserve local ids (where possible), it may make sense to migrate all "site content" entities at once (and not site by site), so soon, we may no longer need to include a separate local site id within the URI. So, in a later PR, this field will likely be removed.